src/api/api.ts

```diff
@@ -0,0 +1,17 @@
+const BASE_URL = 'https://mate.academy/students-api';
+
+export const getTodos = () => {
+  return fetch(`${BASE_URL}/todos`)
+    .then(response => response.json());
```

Comment:
create common method for request handling (call it request)

https://mate-academy.github.io/fe-program/js/extra/fetch#handle-an-error-in-the-component